Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -70,6 +73,9 @@ def boneyard_formula_pr
       end
     end
     branch = "#{formula.name}-boneyard"
+
+    reason = " because #{reason}" if reason
```

Comment:
might be a good idea to make sure the reason string doesn't already start with "because," "since," "due to," etc.